src/Tools.Migration/Microsoft.Graph.Tools.Migration.psd1

```diff
@@ -0,0 +1,134 @@
+#
+# Module manifest for module 'Microsoft.Graph.Tools.Migration'
+#
+# Generated by: Microsoft
+#
+# Generated on: 2/10/2023
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = '.\Microsoft.Graph.Tools.Migration.psm1'
```

Comment:
Should we have `Microsoft.Graph.Tools.Migration` module in a separate GitHub repository? It is currently detached from PowerShell SDK and can be used without a taking a dependency on the SDK. 

cc\ @maisarissi, @ddyett, @timayabi2020

Reply:
> Should we have `Microsoft.Graph.Tools.Migration` module in a separate GitHub repository? It is currently detached from PowerShell SDK and can be used without a taking a dependency on the SDK.
> 
> cc\ @maisarissi, @ddyett, @timayabi2020

I agree. We should have the tool in a separate GitHub repository

Reply:
Sure. If it's detached from the SDK and can be used without taking a dependency, we can have it in a different repository. Also, I believe this is beneficial to the size of this PowerShell module, in case one doesn't need/wanna use the advisor.